content/bdk-cli/installation.md

```diff
@@ -34,59 +34,38 @@ Once Cargo is installed, you can proceed to install the interactive `bdk-cli` to
 the GitHub repository, by running:
 
 ```bash
-cargo install --git https://github.com/bitcoindevkit/bdk-cli --features=esplora
+cargo install --git https://github.com/bitcoindevkit/bdk-cli --features=esplora,compiler,electrum,repl
```

Comment:
You only need to add `compiler` to this list, `electrum` and `repl` are already included as part of the "default" features in the `bdk-cli` Cargo.toml.

Reply:
How about esplora? 
It affects the result in the interface.md where we display the available OPTIONS for `bdk-cli wallet`.

Also wondering if we should add `compact_filters`. My reasoning here is to show all available feature options and since it will be added soon, we could just include it here although it would temporarily break.

Reply:
Also, what is `compiler` used for in bdk-cli?

I've used it when installing bdk to get the minisciptc example binary. Which although a separate issue, I think would be nice to have as part of bdk-cli. The flow to use subcommands would then be key -> compile -> wallet

Reply:
Good point. For all options then you should include esplora and sure I think cbf options are stable now so can add compact_filters too but then we should wait to merge this PR until that PR is in.

Reply:
Never mind the comment on `compiler`, i think i have multiple versions of bdk-cli and i wasnt seeing the `compile` subcommand with `bdk-cli help`. So that flow already exists :P

Reply:
So then, there can be an update to the compiler content as well to use `bdk-cli compile` rather than `miniscriptc`, since it is under the bdk-cli subgroup.

Reply:
It's true the `bdk-cli compile` sub-command will compile a policy to a descriptor, and the `bdk-cli wallet policies` sub-command will display the spending polices which I believe covers everything that the `miniscriptc` example is doing.